packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -108,7 +101,7 @@ export class InsecureUrlResolver implements IUrlResolver {
                 ordererUrl: this.ordererUrl,
                 storageUrl,
             },
-            tokens: { jwt: this.auth(this.tenantId, documentId) },
+            tokens: {},
```

Comment:
Is this field useful any more?

Reply:
We need the field cause you still need a "host token" for talking to Gateway securely. A host may also need to send you "refresh token" for tighter control. Since we already have a token object, we can reuse that easily for providing host specific token. If we can interface those as well (like ITokenProvider), we can get rid of this tokens object permanently.

But our good old "jwt" key for accessing r11s/FRS will be gone soon. There are two local driver usage that I don't quite understand. I will open an issue for that. After that, we can permanently remove the jwt field to start with.

Reply:
We may be able to get rid of this sooner cause we can probably interface the host tokens easily. I would have to check Gateway first. But we need this field for now.

Reply:
Created an issue for the local driver usage #4147